test/easyconfigs/easyconfigs.py

```diff
@@ -642,6 +648,9 @@ def check_python_packages(self, changed_ecs, added_ecs_filenames):
                 if download_dep_fail or exts_download_dep_fail:
                     fail = "'*download_dep_fail' should not be set in %s since PythonBundle easyblock is used" % ec_fn
                     failing_checks.append(fail)
+                if exts_default_options.get('source_urls') == python_default_urls:
+                    failing_checks.append("'source_urls' can be omitted in exts_default_options when using the default"
```

Comment:
`cam be omitted` -> `should be omitted` or `should not be defined` (latter is clearer imho)